src/tasks/harvest_groups.js

```diff
@@ -24,35 +24,34 @@ async function harvest () {
     const allGroups = await harvester.fetchGroups()
     console.log('Total number of groups:', allGroups.meetup.length)
 
-    allGroups.meetup.forEach(item => {
+    for (const item of allGroups.meetup) {
```

Comment:
By using a literal for loop, we can use `await` inside, and thus process each group in sequence, rather than in parallel.